ui/src/wishDetails/WishDetails.js

```diff
@@ -100,7 +103,3 @@ export default class WishDetails extends Component {
     )
   }
 }
-
```

Comment:
Should `id` be added here instead?

Reply:
I am matching it from params instead. We will add proptypes for all components later